tools/prepare.py

```diff
@@ -88,39 +119,46 @@ def load_save(self,
 
         for f in files:
             filename = f.split("/")[-1]
-
-            # load data based the format
+            # load data based on ext, result axis should be in order z, y, x
             if "nii.gz" in filename:
-                f_np = nib.load(f).get_fdata(dtype=np.float32)
+                try:
+                    f_nps = sitk_read(f, split=True)
+                except RuntimeError:
+                    # TODO: nib read 4d series
+                    # TODO: support orient
+                    f_np = nib.load(f).get_fdata(dtype="float32")
+                    f_nps = [f_np.swapaxes(0, 2)]
```

Comment:
These code is a bit abnormal, what is the functionality that you want to achieve?